### DISCUSSION TITLE
只复刻键盘主体，是否可以不需要电池，直接USB供电？

### DISCUSSION THREAD

**Discussion by 0xCC0x01** [peng-zhihui/HelloWord-Keyboard#91] [Q&A]
仅键盘本体，无扩展的情况
是否可以直接由USB提供5V，然后转3.3V？不使用电池

**Comment by xingrz** [comment]
<img width="1241" alt="image" src="https://user-images.githubusercontent.com/288288/185981096-a915f9c5-0ce5-412b-bfc9-8ed3f43772a4.png">

我认为可以。把这块删掉或者不贴，短接VBAT和VBUS。
